src/amplify-migration/yamlUtils.ts

```diff
@@ -0,0 +1,115 @@
+import YAML, { Scalar, isPair, isScalar, isMap, isSeq, Pair } from "yaml";
+import { getRawPermalink } from "./jsDomUtils";
+import { REPOS_WITH_ERRORS } from "./constants";
+import os from "os";
+import fs from "fs";
+import path from "path";
+import { errorMessage } from "./errorMessage";
+
+export const YML_KEYS = [
+  "shareicon",
+  "favicon",
+  "file_url",
+  "image",
+  "logo",
+  "background",
+];
+
+export function isYAMLPair(node: any): node is YAML.Pair<Scalar, Scalar> {
+  if (!isPair(node)) return false;
+  if (isScalar(node.value) && isScalar(node.key)) return true;
+  return false;
+}
```

Comment:
Would be good to leave a comment about what the second item is looking for (that the value is not an array)

Reply:
157470e0f7127dfb7bec1c982eff1c71631001f4
changing func name for this instead